examples/google-signin/google-signin-iframe.html

```diff
@@ -21,5 +21,13 @@
       // This should also include your AMP cache origin(s), if you're using AMP.
       allowedOrigins: ['http://localhost:8000', 'http://localhost:8080'],
     });
+    GaaGoogle3pSignInButton.show({
```

Comment:
Should this be in its own `html` file?

Reply:
Good point -- added new html file and corresponding route for the example!